src/components/Footer/index.js

```diff
@@ -6,7 +6,7 @@ export default  function Footer() {
   return (
     <footer className='footer'>
         <div className="left-side">
-            <h1 className='footer-head'>RaceHunter</h1>
+            <h1 className='footer-head'>Race Hunter</h1>
```

Comment:
```suggestion
            <h1 className='footer-head'>RaceHunter</h1>
```
Επίτηδες δεν έχει κενό. Είναι όπως λέμε GitHub και όχι Git Hub.